make/harbor.yml

```diff
@@ -1,114 +1,91 @@
-ssl_cert_key: /data/cert/server.key
-
-#The path of secretkey storage
-secretkey_path: /data
-
-#Admiral's url, comment this attribute, or set its value to NA when Harbor is standalone
-admiral_url: NA
-
-# Log configurations
-log:
-  # Log files are rotated log_rotate_count times before being removed. If count is 0, old versions are removed rather than rotated.
-  rotate_count: 50
-  # Log files are rotated only if they grow bigger than log_rotate_size bytes. If size is followed by k, the size is assumed to be in kilobytes. 
-  # If the M is used, the size is in megabytes, and if G is used, the size is in gigabytes. So size 100, size 100k, size 100M and size 100G 
-  # are all valid.
-  rotate_size: 200M
-  # The directory that store log files
-  location: /var/log/harbor
-
-#NOTES: The properties between BEGIN INITIAL PROPERTIES and END INITIAL PROPERTIES
-#only take effect in the first boot, the subsequent changes of these properties 
-#should be performed on web ui
-
-##The initial password of Harbor admin, only works for the first time when Harbor starts. 
-#It has no effect after the first launch of Harbor.
-#Change the admin password from UI after launching Harbor.
+# core, harbor
+http:
+  port: 80
+
+# https:
+#   port: 443
+#   #The path of cert and key files for nginx, they are applied only the protocol is set to https
+#   certificate: /your/certificate/path
+#   private_key: /your/private/key/path
+
+# Uncomment extearnal_url if you want to enable external proxy
+# And when it enabled the hostname will no longger used
+# external_url: https://reg.mydomain.com:8433
+
+# The initial password of Harbor admin
+# It only works in first time to install harbor
+# Remember Change the admin password from UI after launching Harbor.
 harbor_admin_password: Harbor12345
 
 ## Harbor DB configuration
 database:
-  #The address of the Harbor database. Only need to change when using external db.
-  host: postgresql
-  #The port of Harbor database host
-  port: 5432
-  #The user name of Harbor database
-  username: postgres
   #The password for the root user of Harbor DB. Change this before any production use.
   password: root123
 
+# The default data volume
+data_volume: /data
 
-# Redis server configuration
-redis:
-  # Redis connection address
-  host: redis
-  # Redis connection port
-  port: 6379
-  # Redis connection password
-  password:
-  # Redis connection db index
-  # db_index 1,2,3 is for registry, jobservice and chartmuseum. 
-  # db_index 0 is for UI, it's unchangeable
-  db_index: 1,2,3
+# Harbor Storage settings
+storage_service:
+  # registry_custom_ca_bundle is the path to the custom root ca certificate, which will be injected into the truststore
+  # of registry's and chart repository's containers.  This is usually needed when the user hosts a internal storage with self signed certificate.
+  ca_bundle:
 
+  # storage backend, default is filesystem, options include filesystem, azure, gcs, s3, swift and oss
```

Comment:
We should explicitly tell user when they choose filesystem, what path will be used.